src/components/CommitHistory/CommitTree/CommitHistory.css

```diff
@@ -37,4 +37,11 @@
 
 .vertical-timeline-element--work:hover{
   cursor: pointer;
+}
+
+.vertical-timeline-element-title {
+  font-family: Montserrat;
```

Comment:
Seems like you did not import it so it does not show any difference, check again.